docs/Change_Log.md

```diff
@@ -1,5 +1,28 @@
 # Change Log
 
+## API v10
+
+#### May 12, 2022
+
+> info
+> API v10 was made available in [Feb 2022](https://github.com/discord/discord-api-docs/discussions/4510). May 12 just indicates the date for this changelog entry.
+
+- API v8 is now deprecated.
+- `GET /channels/{channel.id}/threads/active` is decomissioned in favor of [`GET/guilds/{guild.id}/threads/active`](#DOCS_RESOURCES_GUILD/list-active-threads).
+- Starting in v10, you must specify the message content intent (using `1 << 15`). See more in the [Gateway Intents documentation](#DOCS_TOPICS_GATEWAY/gateway-intents).
```

Comment:
This should mention the affected fields rather than a general statement, as some fields like mentioned roles will still be there despite being part of the message content